src/Shared/Metrics/MetricsConstants.cs

```diff
@@ -11,6 +11,6 @@ internal static class MetricsConstants
     // Not based on a standard. Larger bucket sizes for longer lasting operations, e.g. HTTP connection duration. See https://github.com/open-telemetry/semantic-conventions/issues/336
     public static readonly IReadOnlyList<double> LongSecondsBucketBoundaries = [0.01, 0.02, 0.05, 0.1, 0.2, 0.5, 1, 2, 5, 10, 30, 60, 120, 300];
 
-    // For Blazor/signalR sessions, which can last a long time.
-    public static readonly IReadOnlyList<double> VeryLongSecondsBucketBoundaries = [0.5, 1, 2, 5, 10, 30, 60, 120, 300, 600, 1500, 60*60, 2 * 60 * 60, 4 * 60 * 60];
+    // For blazor circuit sessions, which can last a long time.
+    public static readonly IReadOnlyList<double> VeryLongSecondsBucketBoundaries = [1, 10, 30, 1 * 60, 2 * 60, 3 * 60, 4 * 60, 5 * 60, 6 * 60, 7 * 60, 8 * 60, 9 * 60, 10 * 60, 1 * 60 * 60, 2 * 60 * 60, 3 * 60 * 60, 6 * 60 * 60, 12 * 60 * 60, 24 * 60 * 60];
```

Comment:
These should ideally be exponential eg
1, 3, 10, 30,
1*60, 3*60, 10*60, 30*60,
1*60*60, 3*60*60, 10*60*60, 24*60*60  

This uses an approx multiplier of 3, while meeting user sensible boundaries

Reply:
done

Reply:
Also renamed to be Blazor-specific to `BlazorCircuitSecondsBucketBoundaries`